test/spec/token.js

```diff
@@ -62,6 +62,154 @@ define(function(require) {
       });
     });
 
+    it('returns id_token using sessionToken with issuer', function (done) {
+      return oauthUtil.setupFrame({
+        oktaAuthArgs: {
+          url: 'https://auth-js-test.okta.com',
+          clientId: 'NPSfOkH5eZrTy8PMDlvx',
+          redirectUri: 'https://auth-js-test.okta.com/redirect',
+          issuer: 'https://auth-js-test.okta.com/oauth2/aus8aus76q8iphupD0h7'
+        },
+        getWithoutPromptArgs: {
+          sessionToken: 'testSessionToken'
+        },
+        postMessageSrc: {
+          baseUri: 'https://auth-js-test.okta.com/oauth2/aus8aus76q8iphupD0h7/v1/authorize',
+          queryParams: {
+            'client_id': 'NPSfOkH5eZrTy8PMDlvx',
+            'redirect_uri': 'https://auth-js-test.okta.com/redirect',
+            'response_type': 'id_token',
+            'response_mode': 'okta_post_message',
+            'state': oauthUtil.mockedState,
+            'nonce': oauthUtil.mockedNonce,
+            'scope': 'openid email',
+            'prompt': 'none',
+            'sessionToken': 'testSessionToken'
+          }
+        },
+        postMessageResp: {
+          'id_token': tokens.authServerIdToken,
+          'state': oauthUtil.mockedState
+        },
+        expectedResp: tokens.authServerIdTokenParsed
+      })
+      .fin(function() {
+        done();
```

Comment:
Curious - by doing this, do we swallow exceptions? Or are exceptions exposed in tests (i.e. throwing some exception will result in a failed test).